index.html

```diff
@@ -8,135 +8,172 @@
     <link rel="stylesheet" href="styles/index.css" />
   </head>
 
-  <body class="app-grid">
-    <header class="app-grid__header">
+  <body>
+    <header class="header">
       <h1>Quiz App</h1>
     </header>
 
-    <main class="app-grid__main">
-      <section class="app-grid__main__card">
+    <main class="app-grid">
+      <section class="app-grid__card">
```

Comment:
The class name `app-grid` indicates that this component has something to do with the app-grid. However, this component is not related to the `main`-element. I would suggest to rename this quiz-card component and have it on it's own.
```suggestion
      <section class="card">
        <img ... class="card__bookmark">
```